examples/nest/hpc_benchmark.sli

```diff
@@ -187,27 +187,27 @@ brunel_params dup using
 
 /BuildNetwork
 {
-  
+
   tic % start timer on construction
   % set global kernel parameters
   <<
      /total_num_virtual_procs nvp
      /resolution  dt
      /overwrite_files true
   >> SetKernelStatus
-  
+
   %------------------------- Creation -----------------------------------------
 
   /iaf_psc_alpha    model_params    SetDefaults
-  
+
   /iaf_psc_alpha 1 Create  % To get same spike counter as before NEST 3.0
-  
-  M_INFO (BuildNetwork)  
+
+  M_INFO (BuildNetwork)
   (Creating excitatory population.) message  % show message
   /E_neurons /iaf_psc_alpha NE Create def
-  
+
   /iaf_psc_alpha 1 Create  % To get same spike counter as before NEST 3.0
```

Comment:
Is that relevant for this PR though?

Reply:
I think so. It was added because the benchmark should be comparable with previous versions, but the spike counter is not the same anymore, right? So we do not need it here then, I think it makes the example more confusing now, because the spike counter is not the same anymore. If it is the same, then keep it :)

Reply:
Since`hpc_benchmark.sli` uses Poisson input and thus random numbers, spike counts are expected to be different and there is no reason to do "gymnastics" like this. Just remove it.

Reply:
Done.